entries/w/wp.pl.json

```
@@ -0,0 +1,14 @@
+{
+  "WP": {
```

The full name seems to be Wirtualna Polska.
```suggestion
  "Wirtualna Polska ": {
```